org.kde.klettres.json

```diff
@@ -0,0 +1,33 @@
+{
+    "id": "org.kde.klettres",
+    "branch": "master",
```

Comment:
remove this line - for flathub it must be stable and will automatically set as that by the system.